src/app/routes/index.test.jsx

```diff
@@ -66,13 +65,10 @@ afterEach(() => {
   window.dotcom = undefined;
 });
 
-const getMatchingRoute = pathname =>
-  routes.find(({ path }) =>
-    matchPath(pathname, {
-      path,
-      exact: true,
-    }),
-  );
+const getMatchingRoute = pathname => {
+  const matchingRoutes = matchRoutes(routes, pathname);
+  return matchingRoutes[0].route;
```

Comment:
Is there ever a danger of this not finding a route, and if so, is there a better way of coding against it? The previous solution of using `find` would have returned undefined if there was no matching route...

Reply:
Good call, I've added some optional chaining and defaulting to `500` to handle invalid returns from `matchRoutes`

Reply:
Should it be a 404 rather than a 500?

Reply:
Yes, doh!